db/migrate/20230316041545_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.text :full_name
+      t.text :address
+      t.text :phone
+      t.text :email
+      t.text :password
```

Comment:
Are any of these supposed to be required?

Reply:
on the course im watching, the required is being added on the model, should it be added on the migrations?

Reply:
@djimenez017 yes, otherwise you run the risk null values are inserted via SQL. It's always a good idea to have validation at the database level.